dev/fsw/device.c

```diff
@@ -85,6 +85,16 @@ void fsw_switch_to_dfi_mode(fsw_dev_t *dev)
     dev->mode = FSW_MODE_DFI;
 }
 
+wddr_msr_t fsw_get_current_msr(fsw_dev_t *dev)
```

Comment:
Do we need to pass fsw_dev_t into this function?  It seems like it is always unused.

Reply:
I left it way to match signature at the device level. We typically always pass in the device so I kept that here even if it's unused. I will update to mark parameter as unused.

Reply:
I am ok with leaving as is then, makes sense to pass the device into any device level function call when possible.